packages/lib-utils/src/components/table/VirtualizedTable.tsx

```diff
@@ -23,21 +23,19 @@ export type VirtualizedTableProps<D> = {
   /** Optional load error object. */
   loadError?: LoadError;
   /** Table columns array. */
-  columns: TableColumn<D>[];
+  columns: TableColumn[];
```

Comment:
@fhlavac we will need to confirm if any of these prop updates will impact `hac-infra`.